moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -49,7 +49,8 @@ namespace trajectory_processing
 static const rclcpp::Logger LOGGER =
     rclcpp::get_logger("moveit_trajectory_processing.time_optimal_trajectory_generation");
 
-constexpr double EPS = 0.000001;
+constexpr double DEFAULT_TIMESTEP = 0.001;
```

Comment:
[nit] use `1e-3` to mach the style of the next line.  Also, put this in an anonymous namespace unless it is an exported symbol somewhere?